cdap-app-fabric/src/main/java/co/cask/cdap/scheduler/Scheduler.java

```diff
@@ -41,15 +41,16 @@
    * @param schedule the schedule to add
    * @throws AlreadyExistsException if the schedule already exists
```

Comment:
Actually, I think it's better if we add a ProfileConflictException, otherwise error handling can be difficult when multiple types of ConflictException can be thrown.

Reply:
Added the ProfileConflictException